PlatformIO/load_settings.py

```diff
@@ -53,11 +53,6 @@
 
     env.Append(CPPDEFINES=cpp_defines)
 
```

Comment:
Ok, but platformIO does not give an error or anything?

Reply:
Oh, platformio does give a warning in the log right when the build starts:
```
Verbose mode can be enabled via `-v, --verbose` option

warning: Ignoring attempt to set reserved variable `$TARGETS'
...
```